docs/repositories-configure/managing-branches.md

```diff
@@ -2,6 +2,9 @@
 
 Codacy automatically triggers analysis on the main branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
 
+!!! note
+    We recommend that you set the main branch on Codacy to the default branch of your Git repository. Codacy displays information for the main branch by default on dashboards and the repositories list.
```

Comment:
I am confused with this tip. The dashboard will show the default branch that you have defined. If you set a dev branch as the default on codacy and visit the dashboard that branch will be the one selected, not the main/master 🤔

Reply:
Good point! On the old version of the page, we're calling "main branch" to the default branch selected on Codacy. This becomes quite confusing since as of lately it's also very common to have the default branch on Git repositories called `main`.

On the new version of the page that we're preparing, we're consistently calling it "default branch", which also matches the name on Git and should remove this confusion. :raised_hands:

I'll try to rephrase this in a way that doesn't introduce confusion, but it might be hard. :thinking:

Reply:
Let's wait until we implement [CY-6494](https://codacy.atlassian.net/browse/CY-6494) to make this improvement to the docs. :hourglass: